domain/src/main/java/org/oppia/android/domain/translation/TranslationController.kt

```diff
@@ -67,12 +78,15 @@ class TranslationController @Inject constructor(
   //  different language is selected).
 
   private val dataLock = ReentrantLock()
-  private val appLanguageSettings = mutableMapOf<ProfileId, AppLanguageSelection>()
   private val writtenTranslationLanguageSettings =
     mutableMapOf<ProfileId, WrittenTranslationLanguageSelection>()
   private val audioVoiceoverLanguageSettings =
     mutableMapOf<ProfileId, AudioTranslationLanguageSelection>()
 
+  private val cacheStoreMap =
```

Comment:
This should probably be named based on the type of data being stored (that is, ``AppLanguageSelection``).

Reply:
Yeah makes sense, updated.